immunopepper/mode_build.py

```diff
@@ -458,22 +458,22 @@ def mode_build(arg):
             if (not arg.skip_annotation) and not (arg.libsize_extract):
                 # Build the background
                 logging.info(">>>>>>>>> Start Background processing")
-                with mp.Pool(processes=arg.parallel, initializer=pool_initializer_glob, initargs=(countinfo, genetable, kmer_database)) as pool:
+                with ThreadPool(processes=None, initializer=pool_initializer_glob, initargs=(countinfo, genetable, kmer_database)) as pool:
```

Comment:
If you set `processes` to `None`, this defaults to the number of CPUs on the physical system. This might not be what you want, especially in a shared compute environment. Was there an issue with the previously used `arg.parallel`?

Reply:
Good catch, the main change here was Pool to Threadpool. I will add the right number of processes in the newer commits.